plugins/org.eclipse.fordiac.ide.deployment.opcua/src/org/eclipse/fordiac/ide/deployment/opcua/helpers/Constants.java

```diff
@@ -72,6 +75,9 @@ public class Constants {
 	public static final String FORCE_VALUE = "<Request Action=\"WRITE\"><Connection Destination=\"{0}\" Value=\"{1}\" /></Request>"; //$NON-NLS-1$
 	public static final String CLEAR_FORCE = "<Request Action=\"WRITE\"><Connection Destination=\"{0}\" /></Request>"; //$NON-NLS-1$
 	public static final String QUERY_RESOURCES = "<Request Action=\"QUERY\"></Request>"; //$NON-NLS-1$
+	public static final String QUERY_FB_TYPE = "<Request ID=\"{0}\" Action=\"QUERY\"><FBType Name=\"{1}\" /></Request>"; //$NON-NLS-1$
```

Comment:
The Request ID is not used anymore in this deployment interface. I would suggest to remove it from the messages.
```suggestion
	public static final String QUERY_FB_TYPE = "<Request Action=\"QUERY\"><FBType Name=\"{0}\" /></Request>"; //$NON-NLS-1$
```